pytket/binders/circuit/boxes.cpp

```diff
@@ -76,7 +76,20 @@ void init_boxes(py::module &m) {
           py::arg("circ"))
       .def(
           "get_circuit", [](CircBox &cbox) { return *cbox.to_circuit(); },
-          ":return: the :py:class:`Circuit` described by the box");
+          ":return: the :py:class:`Circuit` described by the box")
+      .def(
+          "symbol_substitution",
```

Comment:
It may be worth noting here that if a `CircBox` has been added to a circuit before this method is called, calling this method affects the circuit it has been added to. I can see this being a trip hazard.

Reply:
I agree it seems like a hazard. Should we try to avoid this behaviour? (Should `add_circbox` add a copy of the circbox to the circuit?)